integreat_cms/cms/templates/analytics/_page_hix_widget.html

```diff
@@ -0,0 +1,40 @@
+{% extends "_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load rules %}
+{% load content_filters %}
+{% block collapsible_box_icon %}
+    gauge
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Overview of the text understandability of the pages" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div class="flex flex-wrap justify-between gap-2">
+        <p>{% translate "The list shows pages by HIX value in ascending order." %}</p>
+        <p>
+            <b>{{ ready_for_mt }}%</b> {% translate " of all pages are ready for automatic translation." %}
+        </p>
+        <p class="text-red-500">
+            {% translate "To be able to machine translate pages, the page must have a HIX score of at least 15. If the row is marked red, the HIX value of the corresponding page is not sufficient." %}
+        </p>
+    </div>
+    <div class="mt-4">
+        <table id="page_hix_list" class="w-full max-w-xl mb-4">
+            <thead>
+                <tr>
+                    <th class="text-sm text-left uppercase py-3 pr-2">{% translate "Page name" %}</th>
+                    <th class="text-sm text-left uppercase py-3 pr-2">{% translate "HIX value" %}</th>
```

Comment:
```suggestion
                <tr class="border-b border-gray-400">
                    <th class="text-sm text-left uppercase p-1">{% translate "Page name" %}</th>
                    <th class="text-sm text-left uppercase p-1">{% translate "HIX value" %}</th>
```
What would you think about this styling?